### ISSUE TITLE
Android Full-screen mode (immersive mode)

### ISSUE THREAD

**Issue by adrach** [apache/cordova-plugin-inappbrowser#380]
Hi all,

Is it possible to make in-app-browser show in full-screen without the system status bar?  
I enabled immersive mode `this.androidFullScreen.immersiveMode()` and launch in-app-browser with `fullscreen: yes` but the system status bar becomes visible as soon as in-app-browser is open

any ideas?

**Comment by janpio** [comment]
> this.androidFullScreen.immersiveMode()

Where does this come from? Is this from another plugin?

**Comment by janpio** [comment]
You should probably ask in their Github issues if and how they support InAppBrowser.

Besides that, you could look into https://github.com/apache/cordova-plugin-statusbar if this can give you the results you want.

**Comment by adrach** [comment]
I tried using `cordova-plugin-statusbar` and those settings also only apply to the main window, and do not affect the in-app-browser modal/window

**Comment by adrach** [comment]
nope, no luck yet. I opened a ticket in both repos

**Comment by aahedi** [comment]
Hi adrach,
I have exactly the same problem.
Did you discover how to solve it?
Thanks,

**Comment by gastondoa** [comment]
Hi aahedi!

  For my part, I have not find any workaround.

**Comment by aahedi** [comment]
I have reference for contributors https://stackoverflow.com/questions/36602322/cordova-inappbrowser-full-screen-android

**Comment by tonetechnician** [comment]
Yeah, I'm also running into this problem. I've tried a variety of things as per the @aahedi's reference and this reference -> https://stackoverflow.com/questions/26259361/apache-cordova-inappbrowser-fullscreen-mode.

I've also tried with StatusBar.hide() to no avail.

All seem really old. We need a modern solution! Unsure exactly what could be causing this issue.

**Comment by aahedi** [comment]
I have reference for contributors https://stackoverflow.com/a/49922293
